influxdb/client.py

```diff
@@ -410,6 +410,18 @@ def write_points(self,
                                       retention_policy=retention_policy,
                                       tags=tags, protocol=protocol)
 
+    def ping(self):
+        """Check connectivity to influxdb
```

Comment:
Minor thing, but summary should end in a period and `influxdb` replaced with `InfluxDB`, so that it will be in the same style as other docstrings. Also, there should be a blank line after the summary line (PEP 257). Otherwise, having this method would be really useful in my opinion 👍